docs/source/workflow_outputs.rst

```diff
@@ -319,11 +319,16 @@ Annual unmet hours are listed below.
   =========================  =====
   Type                       Notes
   =========================  =====
-  Unmet Hours: Heating (hr)  Number of hours where the heating setpoint is not maintained.
+  Unmet Hours: Heating (hr)  Number of hours where the heating setpoint is not maintained. [#]_
   Unmet Hours: Cooling (hr)  Number of hours where the cooling setpoint is not maintained.
+  Unmet Hours: Showers (hr)  Number of hours where the shower water temperature is less than 105 deg-F. [#]_
```

Comment:
I discussed with @jmaguire1; this should really say water heater tank outlet temperature, not shower water temperature. I will change this.